### ISSUE TITLE
Copyright for pages?

### ISSUE THREAD

**Issue by jvoight** [LMFDB/lmfdb#5088]
We have had some questions about to what extent LMFDB pages are copyrighted.  This is analogous I guess to Wikipedia.  Probably that means the whole site (images, webpages, ...) should be covered by CC-BY-NC-SA?

**Comment by roed314** [comment]
It would be good to clarify.  I'm not a lawyer (obviously), but I think that copyright already exists and is held by the people who have contributed.  I think a site-wide license would be helpful; there are arguments in favor of various creative commons licenses and we should check in with the current contributors to see what they would be comfortable with.  In particular, I think we need to get EVERYONE'S permission to add such a site license.

**Comment by davidfarmer** [comment]
(I wrote this comment in an email, but don't see it here.)

I suggest CC-BY-SA.

That is, omit the NC .

People seem to like the NC because of a belief that it protects you from being exploited by an evil commercial publisher.

But:

1) SA probably gives you all the protection you seek.

2) Probably NC prevents a lot of usage which you would consider
legitimate.

**Comment by davidfarmer** [comment]
I suggest CC-BY-SA.

That is, omit the NC .

People seem to like the NC because of a belief that it protects
you from being exploited by an evil commercial publisher.

But:

1) SA probably gives you all the protection you seek.

2) Probably NC prevents a lot of usage which you would consider
legitimate.

**Comment by jvoight** [comment]
Sounds reasonable, what are some examples of 2 (where NC would prevent legitimate usage)?

**Comment by davidfarmer** [comment]
Can a professor have the campus bookstore include that material in a course
pack?  Seems like commercial usage to me, because the bookstore is making
money from it.

Can someone put that material in a document (say, a textbook) they want to
release under a CC BY-SA license (the most common for open textbooks)?
No, because the licenses are incompatible.

And why did my email of March 20 just appear in the issue today?

**Comment by jvoight** [comment]
Let's decide this during the LMFDB meeting in July (at LuCaNT).

**Comment by roed314** [comment]
I don't know that this actually got discussed at LuCaNT....

**Comment by jvoight** [comment]
Yes, Drew and I discussed it and decided not to bring it up.  It wasn't clear that it should be a priority, we weren't sure we had all of the information yet or if we'd need actual legal advice, and this type of topic tends to engender a lot of conversation.  I think kicking the can is a good idea here.
